scripts/get_all_heron_paths.sh

```diff
@@ -23,7 +23,8 @@ function query() {
 
 set +e
 # Build everything
-bazel build {heron,integration-test,tools/java}/...
+source ./detect_os_type.sh
```

Comment:
Hi @billonahill 
relative path of ```detect_os_type.sh``` here is not indeed. 
I have fixed it. Thank you for pointing out :)